commons/commons-test/src/main/kotlin/ebi/ac/uk/await/ConditioanlFactoryExt.kt

```diff
@@ -0,0 +1,12 @@
+package ebi.ac.uk.await
+
+import org.awaitility.core.ConditionFactory
+
+fun <T> ConditionFactory.untilNotNull(func: () -> T): T {
```

Comment:
if not wait time is provider library use `defaultWaitConstraint` (ten seconds) when you call `await` so in this case library is safe to prevent this scenario.